### DISCUSSION TITLE
any estimate for the upcoming release that is compatible with v1.35 (vasil hard fork)?

### DISCUSSION THREAD

**Discussion by monkey-jsun** [cardano-foundation/cardano-wallet#3356] [General]
Would appreciate an estimate so that I can plan ahead.  Thanks!

**Comment by HeinrichApfelmus** [comment]
One year later, I can say with confidence that `cardano-wallet` [release v2022-07-01](https://github.com/cardano-foundation/cardano-wallet/releases/tag/v2022-07-01) was the first release to be compatible with `cardano-node`  v1.35.
